template/config.sls

```diff
@@ -10,3 +10,17 @@ template-config:
     - mode: 644
     - user: root
     - group: root
+
+# If don't you have other config files to manage, remove these lines
+# The parameter `manage_another_config must default to False` in
+# defaults.yaml
```

Comment:
Maybe we could go ahead and simply add those parameters (`manage_another_config`, `another_config`) to the `defaults.yaml` file? :smile:

Reply:
I'm not sure if that is a good idea, this basically assumes that someone will use the template to create a formula with optional configuration.